digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/service/lane/LaneService.scala

```diff
@@ -106,6 +108,18 @@ trait LaneOperations {
     getSegmentedViewOnlyLanes(allLanes, filteredRoadLinks)
   }
 
+  def getLaneWorkList(): Seq[LaneWorkListItem] = {
```

Comment:
Hyym näille ehkä olis voinut tehdä oman service, ei nyt akuuttia mut nää ei nyt suoranaisesti enään liity kaistojen ydin operaatioihin.

Reply:
> Kai tällä nyt voidaan mennä eteenpäin mutta olis kyl hyvä korja jossakin toisessa vaiheessa. Voi tehdä nytkin jos ei ole suurta kiirettä. -- pääpoint Kyl tällä nyt voidaan mennä mutta omasta mielessä olis pitänyt tehdä akka actor näille.
> 
> Tielinkien ei tarvii tietää mitään kaistoista. Nyt alkaa tulla vaara että kaistat ja tielinkkit menee ristiin. Vähän samaa on jo tapahtunut noissa RoadAddress hommissa.

Varmaan voisi mennä tällä nyt eteenpäin, niin saa tuotantojulkaisun käyntiin pikimmiten. Actor käyttö olisi kyllä ollut parempi ratkaisu.